deployments/kubernetes/chart/ingressmonitorcontroller/templates/configmap.yaml

```diff
@@ -19,6 +19,7 @@ data:
     - name: {{ .name }}
       apiKey: {{ .apiKey }}
       apiURL: {{ .apiURL }}
+      username: {{ .username }}
```

Comment:
Can you make this optional in the chart? Since some providers don't need username